src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -51,6 +41,49 @@ <h1>{{ title }}</h1>
     - could be a mistake
```

Comment:
In public facing communication, we should use complete sentences. How about the following: 

_The provided code is invalid, unauthorized, expired or incomplete. Please use the link in the email sent to your email address {{ email }}._

If you'd like to keep the list format, it should start with something like "It could be". Still I would move the CTA below the list, then.

Reply:
@@flodolo requested the list format as a more readable version. I'm not sure why we shouldn't be allowed to use bullet lists also.

Reply:
Updated to sentence.